auth-api/requirements.txt

```diff
@@ -9,7 +9,7 @@ Mako==1.1.3
 MarkupSafe==1.1.1
 SQLAlchemy-Continuum==1.3.11
 SQLAlchemy-Utils==0.36.8
-SQLAlchemy==1.3.19
+SQLAlchemy==1.3.20
```

Comment:
are these upgrades in requirements.txt are ok?

@sumesh-aot @saravanpa-aot

Reply:
yup